module/VuFind/tests/unit-tests/src/VuFindTest/CartTest.php

```diff
@@ -47,7 +48,7 @@ class CartTest extends \PHPUnit\Framework\TestCase
     /**
      * Mock record loader
      *
-     * @param \VuFind\Record\Loader
+     * @var MockObject
```

Comment:
I think it's probably more informative to typehint this as a \VuFind\Record\Loader than as a MockObject. Of course, best option might be `\VuFind\Record\Loader&MockObject`, though I know some static analysis tools have trouble with that.

Reply:
Looks like phpstan does not have problem with this, so I tried it this way.